app/services/xccdf_report_parser.rb

```diff
@@ -36,7 +36,7 @@ def initialize(report_contents, message)
     end
 
     @b64_identity = message['b64_identity']
-    @account = Account.find_or_create_by(account_number: message['account'])
+    @account = Account.from_identity_header(IdentityHeader.new(@b64_identity))
```

Comment:
Here, `@b64_identity` is always `nil`. It's never been used in this parser before this change, so we never detected it as an issue.